Sources/Components/TableViewCells/Demo/TableViewCellsDemoView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+fileprivate enum TableViewCellType: Int {
+    case basic, checkbox, checkboxSubtitle
+}
+
+protocol HapticFeedbackProducer: AnyObject {
```

Comment:
Why using a protocol for this?

Reply:
Protocols are cheap, and I wanted a single place to change/test how much feedback a selection should produce.